lib/routes/account.js

```diff
@@ -204,7 +204,8 @@ module.exports = function (
             email: account.email,
             emailCode: account.emailCode,
             emailVerified: account.emailVerified,
-            verifierSetAt: account.verifierSetAt
+            verifierSetAt: account.verifierSetAt,
+            createdAt: parseInt(request.query.createdAt)
```

Comment:
Good point, would it be clearer if I moved the call to `parseInt` later on, where the `>= 0` check is made?